Frameworks/CoreFoundation/URL.subproj/CFURL.c

```diff
@@ -1387,32 +1396,32 @@ CF_EXPORT CFStringRef CFURLCreateStringByAddingPercentEscapes(CFAllocatorRef all
             }
             else {
                 // FIXME: once CFString supports finding glyph boundaries walk by glyph boundaries instead of by unichars
-                if ( encoding == kCFStringEncodingUTF8 && CFCharacterSetIsSurrogateHighCharacter(ch) && idx + 1 < length && CFCharacterSetIsSurrogateLowCharacter(__CFStringGetCharacterFromInlineBufferQuick(&buf, idx+1)) ) {
+                if ( encoding == kCFStringEncodingUTF8 && CFCharacterSetIsSurrogateHighCharacter(ch) && idx + 1 < length && CFCharacterSetIsSurrogateLowCharacter(__CFStringGetCharacterFromInlineBufferQuick(&buf, idx + 1)) ) {
                     UniChar surrogate[2];
                     uint8_t *currByte;
-                    
+
                     surrogate[0] = ch;
-                    surrogate[1] = __CFStringGetCharacterFromInlineBufferQuick(&buf, idx+1);
+                    surrogate[1] = __CFStringGetCharacterFromInlineBufferQuick(&buf, idx + 1);
                     // Aki sez it should never take more than 6 bytes
                     if (CFStringEncodingUnicodeToBytes(kCFStringEncodingUTF8, 0, surrogate, 2, NULL, bytes, 6, &byteLength) == kCFStringEncodingConversionSuccess) {
                         endPtr = bytePtr + byteLength;
-                        for ( currByte = bytes; currByte < endPtr; currByte++ ) {
+                        for (currByte = bytes; currByte < endPtr; currByte++) {
                             charBuffer[charBufferIndex++] = '%';
                             charBuffer[charBufferIndex++] = hexchars[*currByte >> 4];
                             charBuffer[charBufferIndex++] = hexchars[*currByte & 0x0f];
                         }
                         idx++; // We consumed 2 characters, not 1
-                    }
-                    else {
+                    } else {
```

Comment:
also 1424-1436

---
In reply to: [77097021](https://github.com/Microsoft/WinObjC/pull/946#discussion_r77097021) [](ancestors = 77097021)